tests/Mover.test.tsx

```diff
@@ -3145,3 +3146,366 @@ describe("Mover with virtual children provided by getParent()", () => {
             });
     });
 });
+
+describe("Mover connected with other Movers", () => {
+    beforeEach(async () => {
+        await BroTest.bootstrapTabsterPage({ mover: true, groupper: true });
+    });
+
+    it("should move between connected Movers", async () => {
+        await new BroTest.BroTest(
+            (
+                <div {...getTabsterAttribute({ root: {} })}>
+                    <div
+                        {...getTabsterAttribute({
+                            mover: {
+                                direction: MoverDirections.Horizontal,
+                                connected: MoverConnections.Child,
+                                cyclic: true,
+                                memorizeCurrent: true,
+                            },
+                        })}
+                    >
+                        <div
+                            tabIndex={0}
+                            {...getTabsterAttribute({
+                                groupper: {},
+                                mover: {
+                                    direction: MoverDirections.Vertical,
+                                    connected: MoverConnections.Parent,
```

Comment:
why are `Child` and `Parent` relationships both needed for connected movers? Is there a scenario where you can have a parent mover with `Child` and only a subset of movers will use `Parent`?